osu.Game.Rulesets.Osu/UI/OsuPlayfield.cs

```diff
@@ -69,10 +72,13 @@ private void onNewResult(DrawableHitObject judgedObject, JudgementResult result)
             if (!judgedObject.DisplayResult || !DisplayJudgements)
                 return;
 
+            var explosionBaseSize = rulesetContainer.Beatmap.BeatmapInfo.BaseDifficulty.CircleSize;
+
             DrawableOsuJudgement explosion = new DrawableOsuJudgement(result, judgedObject)
             {
                 Origin = Anchor.Centre,
-                Position = ((OsuHitObject)judgedObject.HitObject).StackedEndPosition
+                Position = ((OsuHitObject)judgedObject.HitObject).StackedEndPosition,
+                Scale = new Vector2((1.0f - 0.7f * (explosionBaseSize - 5) / 5) / 2 * 1.65f)
```

Comment:
This should probably just come from the `judgedObject`'s scale?

Reply:
Sometimes it comes from the slider which always has a (1, 1) scale

Reply:
see `OsuHitObject.Scale`. ie. `judgedObject.HitObject.Scale`